<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>714</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="TabSubSystem">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="TabMainSystem">
       <attribute name="title">
        <string>Main</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="GuiBlocks::View" name="MainView"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Test Inputs</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0,0,0">
       <item>
        <widget class="QPushButton" name="btnAddItem">
         <property name="text">
          <string>Add Item</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnFilpH">
         <property name="text">
          <string>Filp H</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnForceC">
         <property name="text">
          <string>Force Connected</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>714</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GuiBlocks::View</class>
   <extends>QGraphicsView</extends>
   <header>GuiBlocks/View.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
